mq/mq.go

```diff
@@ -34,8 +34,8 @@ func setMqOptions(user, password string, opts *mqtt.ClientOptions) {
 	opts.SetAutoReconnect(true)
 	opts.SetConnectRetry(true)
 	opts.SetCleanSession(true)
-	opts.SetConnectRetryInterval(time.Second * 4)
-	opts.SetKeepAlive(time.Minute)
+	opts.SetConnectRetryInterval(time.Second * 3)
+	opts.SetKeepAlive(time.Second * 5)
```

Comment:
@yabinma do you think it's fine to change the `SetKeepAlive` back to a minute?
I am just thinking if this will cause too many ping requests in a given time at Scale on SaaS

Reply:
This is not a big issue as I can see. There is only one server connecting to mq, the traffic should not be a problem.
I will change it back to 1 minute in next commit.   Changing to 5 seconds does not help in the mq issue we met. Also in emqx settings, the server side time settings is matched with 1 minute.